src/objecttypes/api/models.py

```diff
@@ -15,6 +17,35 @@ class ObjectType(models.Model):
     name_plural = models.CharField(
         _("name plural"), max_length=100, help_text="Plural name of the object type"
     )
+    description = models.CharField(
+        _("description"),
+        max_length=1000,
+        blank=True,
+        help_text="The description of the object type",
+    )
+    public = models.BooleanField(
+        _("public"),
+        default=True,
+        help_text="Indicates whether this data is accessible without any specific authorizations",
+    )
+    maintainer = models.CharField(
+        _("maintainer"),
+        max_length=200,
+        blank=True,
+        help_text="Organization which is responsible for the object type",
+    )
+    contact = models.CharField(
```

Comment:
Lets call/rework this as "maintainerContactEmail" to prevent storing personal data on a public dataset.